lib/guest_installation_and_configuration_base.pm

```diff
@@ -92,11 +92,10 @@ our %guest_params = (
     'guest_network_others' => '', #virt-install --netowrk=bridge=[guest_network_device],mac=[guest_macaddr],[guest_network_others]
                                   #(Also can be used with other network type)
     'guest_macaddr'        => '', #virt-install --network=bridge=[guest_network_device],mac=[guest_macaddr] (Also can be used with other network type)
-    'guest_netaddr'        => '', #This indicates the subnet to which guest will be connected. It takes the form ip_address/subnet_mask_length and
-        #defaults to 192.168.123.255/24, not virt-install argument. If 'host-default' is given, this indicates guest will use host network
-        #and host bridge device that already exists and are connected directly to default gateway, for example, br0. If br0 or any other
-        #host bridge devices already conneced to host network that do not exist, [guest_network_device] wil be configured to connect to host
-        #network and used for guest configuration.
+    'guest_netaddr' => '', #This indicates the subnet to which guest will be connected. It takes the form ip_address/subnet_mask_length and defaults to 192.168.123.255/24,
```

Comment:
"=>" is not aligned

Reply:
It depends what CI instructs me to do. CI will not let me pass if align according to your own standard.

Reply:
Okay, if CI forces you to do so. I remember we used to point out such CI align rule issue and oliver gave some suggestion about workaround for it in one of your previous PR.

Reply:
Okay, if CI forces you to do so. I remember we used to point out such CI align rule issue and oliver gave some suggestion about workaround for it in one of your previous PR.

Reply:
I always try to make things tidy. But the CI has the final say.